htdocs/core/actions_massactions.inc.php

```diff
@@ -1388,6 +1388,7 @@
 	foreach ($toselect as $toselectid) {
 		$result = $objecttmp->fetch($toselectid);
 		if ($result > 0) {
+			$objecttmp->fetch_thirdparty(); //load lang from thirdparty
```

Comment:
I suggest to load the thirdparty only if we need it so inside the test line 1398.
what do you think ?

Reply:
Unfortunately it does not enter the IF

Reply:
@eldy I did a second version, what do you think